frontend/src/app/components/containers/complaints/outcomes/animal-outcomes/animal-outcome-item.tsx

```diff
@@ -16,7 +16,7 @@ import { selectOfficersByAgencyDropdown } from "../../../../../store/reducers/of
 import { formatDate, getAvatarInitials } from "../../../../../common/methods";
 import { from } from "linq-to-typescript";
 import { DrugItem } from "./drug-item";
-import { BsPencil } from "react-icons/bs";
+import { BsPencil, BsTrash } from "react-icons/bs";
```

Comment:
I think the BsTrash3 looks more exact in the design

Reply:
Agree